src/SFA.DAS.EmployerIncentives.Api/HealthChecks/InnerApiHealthCheck.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using SFA.DAS.EmployerIncentives.Interfaces;
+
+namespace SFA.DAS.EmployerIncentives.Api.HealthChecks
+{
+    public class InnerApiHealthCheck : IHealthCheck
```

Comment:
This isn't used and can be deleted - EmployerIncentivesHealthCheck is what is registered in Startup

Reply:
deleted